### PR TITLE
adding_example_improving_docs

### LINKED ISSUE #34: Updates examples
New example is desired to show the user how to add two or more sensors

### PR DESCRIPTION
**Pull request by jposada202020**
close #34 
Included the new example as discussed in the issue.

Other Docs improvements
- Sphinx inter mapping: remove reference to register library as it is not required
- Add example to the API
- Add Learning Guide
- Add black badge
- Add Bus_register and blinka to requirements and setup.py
- Typos and other minor changes

### CODE REVIEW

**Comment by jposada202020** [comment]
@caternuson as discussed, let me know if this is ok. thanks

File: README.rst [hunk 1]
```diff
@@ -56,28 +61,22 @@ Usage Example
 
 .. code-block :: python
 
-    # This example shows using two TSL2491 light sensors attached to TCA9548A channels 0 and 1.
-    # Use with other I2C sensors would be similar.
-    import time
+    # This example shows using sensors attached to TCA9548A channels 1 and 5.
```
Comment (caternuson) [inline README.rst]: Example looks like a scan. Need comment change here?

File: examples/tca9548a_simpletest.py [hunk 2]
```diff
@@ -1,25 +1,19 @@
-# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-FileCopyrightText: 2021 Carter Nelson for Adafruit Industries
 # SPDX-License-Identifier: MIT
 
-# This example shows using two TSL2491 light sensors attached to TCA9548A channels 0 and 1.
-# Use with other I2C sensors would be similar.
-import time
+# This example shows using sensors attached to TCA9548A channels 1 and 5.
```
Comment (caternuson) [inline examples/tca9548a_simpletest.py]: Same here.

**Review by caternuson — CHANGES_REQUESTED** [review]
Looks good. Just a question on comment line(s).

File: examples/tca9548a_simpletest.py (hunk 2 above)
Comment (jposada202020) [reply examples/tca9548a_simpletest.py]: Solved in https://github.com/adafruit/Adafruit_CircuitPython_TCA9548A/pull/35/commits/8d66db73557f009d38d79dfca13f7e8d0e8e88ff
Thanks

File: README.rst (hunk 1 above)
Comment (jposada202020) [reply README.rst]: Solved in https://github.com/adafruit/Adafruit_CircuitPython_TCA9548A/pull/35/commits/8d66db73557f009d38d79dfca13f7e8d0e8e88ff
Thanks

**Comment by jposada202020** [comment]
@caternuson I changed the comments as suggested. Thanks.

**Review by caternuson — APPROVED** [review]

**Comment by evaherrada** [comment]
@jposada202020 Do you think you could reopen this branch and make another PR? I messed something up with moving to main and this PR wasn't included in the main branch.

**Comment by tannewt** [comment]
Hold off on making another PR. We should be able to recover the full history.

**Pull request merged**